Frameworks/UIKit/NSTextContainer.mm

```diff
@@ -62,38 +78,120 @@ - (NSArray*)exclusionPaths {
     return _exclusionPaths;
 }
 
+// Finds the first viable region for the proposed rect given the exclusion areas based upon x positions
+// Assumes writing direction is left to right, exclusionLines is sorted and preprocessed for y positions, and all inputs are valid
```

Comment:
>Assumes writing direction is left to right [](start = 3, length = 42)

What's the plan for eventually dealing with this? It's not clear to me atm whether this section will be removed/fixed once CGD2D comes in